spec/fixtures/constants.rb

```diff
@@ -106,6 +106,13 @@
     cd /Users/musalj/code/OSS/Alcatraz
     /usr/bin/touch -c /Users/musalj/Library/Developer/Xcode/DerivedData/Alcatraz-aobuxcinaqyzjugrnxjjhfzgwaou/Build/Products/Debug/AlcatrazTests.octest
 )
+
+SAMPLE_WRITE_FILE = %Q(
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.